tools/test-proxy/documentation/asset-sync/README.md

```diff
@@ -42,6 +42,17 @@ As one can see in the example image above, the test-proxy does the heavy lifting
 
 The `Tag` "commit SHA" is literally the SHA of the tag being pushed. This allows us limited restore capabilities in the case of non-GC-ed accidentally-deleted tags.
 
+### How does the test-proxy relate to the `assets.json`?
+
+The `assets.json` contains _targeting_ information about WHERE to get recordings, but how do those recordings actually end up available on disk?
+
+The test-proxy will restore an `assets.json` file in two scenarios:
```

Comment:
When you say the proxy will "restore an `assets.json` file", do you mean that it's using the `assets.json` file for recording operations? The phrase here makes me think that the `assets.json` file itself is being fetched or restored from some data, but this section is about how the recordings are managed.